src/app/PublicationArticles/page.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+
+import styles from './styles.module.css';
+
+const page = () => {
+  return (
+    <div className={styles.container}>
+        <div>
+            <button className={styles.button}><Link href="/PublicationArticles/NewArticulo">Nuevo Articulo</Link> </button>
+            <svg xmlns="http://www.w3.org/2000/svg" width="93" height="88" viewBox="0 0 93 88" fill="none" className={styles.svgimage}>
+                <g clip-path="url(#clip0_1_8)">
+                <path d="M0 30.4693C0.357832 29.0937 0.880957 27.7785 2.1252 26.9974C2.87537 26.5273 3.75088 26.0664 4.60277 25.9914C6.36832 25.8377 8.15748 25.9456 9.9975 25.9456V30.3998C9.01482 30.3998 8.08301 30.387 7.15301 30.4016C5.3584 30.4309 4.48471 31.3016 4.48289 33.1217C4.47744 49.0011 4.48107 64.8824 4.48834 80.7618C4.48834 82.5581 5.45285 83.4983 7.25473 83.4965C8.2156 83.4965 9.36357 83.7599 10.0811 83.3245C10.764 82.9111 11.0074 81.7478 11.4216 80.9045C16.0861 71.4127 20.7379 61.9136 25.4224 52.4309C25.8475 51.5712 26.4214 50.7645 27.0354 50.0273C27.92 48.9645 29.1097 48.5548 30.5011 48.5566C42.6383 48.5749 54.7755 48.5676 66.9128 48.5676H68.137V35.8911C70.7109 36.0283 72.5818 38.0642 72.6345 40.7221C72.678 42.9775 72.6454 45.2329 72.6472 47.4883C72.6472 47.8158 72.6472 48.1432 72.6472 48.5676H73.7061C78.852 48.5676 83.9979 48.6133 89.142 48.542C90.873 48.5182 92.2044 48.9663 93 50.5889V52.0522C92.466 53.1717 91.9628 54.3077 91.3907 55.407C89.4017 59.2374 87.3873 63.055 85.382 66.8762C82.3177 72.7169 79.2771 78.5704 76.1764 84.391C75.0376 86.5312 73.3083 87.9543 70.7763 87.9561C49.0738 87.9817 27.3732 87.9817 5.67082 87.9964C3.73271 87.9964 2.09432 87.3616 0.97541 85.7226C0.557637 85.1117 0.319687 84.3745 0 83.6959C0 65.9525 0 48.2091 0 30.4656L0 30.4693Z" fill="#01461D"/>
```

Comment:
don't use the SVG embedded directly within the components, you have to call them from the files in this way.
```
import myIcon from '../../somerute/icon.svg'
```